src/contracts/pods/EigenPod.sol

```diff
@@ -697,9 +701,12 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
          * Finally, the validator is fully withdrawn. Update their status and place in state:
          */
 
-        validatorInfo.restakedBalanceGwei = 0;
-        validatorInfo.status = VALIDATOR_STATUS.WITHDRAWN;
+        if (validatorInfo.status != VALIDATOR_STATUS.WITHDRAWN) {
```

Comment:
this is equivalent to `if (validatorInfo.status == VALIDATOR_STATUS.ACTIVE)`, because we require that the status is not INACTIVE above.
seems OK 👍